samples/subsys/usb/common/sample_usbd.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
This file is missing include guards.

Reply:
Then it is more pain than just guarding.